frontend/package.json

```diff
@@ -59,6 +59,7 @@
     "@textea/json-viewer": "^3.2.2",
     "@types/js-cookie": "^3.0.5",
     "@types/react-grid-layout": "^1.3.4",
+    "@uiw/react-codemirror": "^4.21.19",
```

Comment:
this basically binds codemirror to react. we do this for the cell's editors ourselves, but it can get messy so rather just use a library instead of do it again or merge those 2 codepaths